tor/controller_test.go

```diff
@@ -1,10 +1,12 @@
 package tor
 
 import (
+	"fmt"
 	"net"
 	"net/textproto"
 	"testing"
 
+	"github.com/lightningnetwork/lnd/lntest/port"
```

Comment:
We're in a submodule and don't have this dependency available (it would create a circular one if we did).
Perhaps we can just copy the `NextAvailablePort` function over? A bit of duplication doesn't sound too bad here, especially if it's just test code.

Reply:
yeah right - guess i'm using go work locally so it never popped up...

Reply:
actually i think in order to run the new test we need to tag the module `tor`?

Reply:
The unit test will run in `make unit-module`. But yes, for the `unit cover` flake to be fixed we need to tag and update the `tor` submodule.